	Posix requires to include <sys/wait.h>, not <wait.h>.
	FreeBSD has no <wait.h>.

	From Nikola Lecic <nikola.lecic@anthesphoria.net>.

diff -ur dvipng-1.14.orig/special.c dvipng-1.14/special.c
--- dvipng-1.14.orig/special.c	2010-12-15 16:01:07.000000000 +0100
+++ dvipng-1.14/special.c	2011-02-07 11:47:08.000000000 +0100
@@ -26,7 +26,7 @@
 
 #ifndef MIKTEX
 #ifndef WIN32
-#include <wait.h>
+#include <sys/wait.h>
 #else /* WIN32 */
 #include <fcntl.h>
 #include <io.h>
